keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -0,0 +1,451 @@
+
+- `TopologyManagerPolicyAlphaOptions` is not enabled by default. Topology Manager alpha options (only one as of 1.26), are hidden by default
+ and only when `TopologyManagerPolicyAlphaOptions` feature gate is enabled user is able to use alpha options.
+  `TopologyManagerPolicyAlphaOptions` will not exist if there are no options in alpha stage.
+
+- `TopologyManagerPolicyBetaOptions` is not enabled by default. Topology Manager beta options (none as of 1.26), are hidden by default
+ and only when `TopologyManagerPolicyBetaOptions` feature gate is enabled user is able to use beta options.
+`TopologyManagerPolicyBetaOptions` will not exist if there are no options in beta stage (true for 1.26 release).
+
+- `TopologyManagerPolicyOptions` for enabling/disabling the entire feature. As this is an alpha feature, this feature gate would be disabled by default.
+Explicitly enabling `TopologyManagerPolicyOptions` feature gate provides us the ability to supply `TopologyManagerPolicyOptions` or `topology-manager-policy-options` flag in Kubelet.
+
+When an option graduates, its visibility should be moved to be controlled by the corresponding feature-flag.
+The introduction of these feature gates gives us the ability to move the option to beta and later stable without implying that all available options are stable.
+
+The graduation Criteria of options is described below:
+
+#### Graduation of Options to `Beta-quality` (non-hidden)
+- [X] Gather feedback from the consumer of the policy option.
+- [X] No major bugs reported in the previous cycle.
+
+#### Graduation of Options from `Beta-quality` to `G.A-quality`
+- [X] Allowing time for feedback (1 year) on the policy option.
+- [X] Risks have been addressed.
+
+
+### Upgrade / Downgrade Strategy
+
+We expect no impact. The new policiy options are opt-in.
+
+### Version Skew Strategy
+
+No changes needed
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate names:
+    - `TopologyManagerPolicyAlphaOptions`
+    - `TopologyManagerPolicyBetaOptions`
+    - `TopologyManagerPolicyOptions`
+  - Components depending on the feature gate: kubelet
+- [x] Change the kubelet configuration to set a `TopologyManager` policy to `restricted` or `best-effort` and a `TopologyManagerPolicyOptions` to `prefer-closest-numa-nodes`
+  - Will enabling / disabling the feature require downtime of the control
+    plane? 
+    No.
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+    Yes, a kubelet restart is required.
+
+###### Does enabling the feature change any default behavior?
+
+- Yes, it makes the behaviour of the TopologyManager restricted and best-effort policies to choose NUMA nodes based on distance.
```

Comment:
I'd argue the answer here is actually 'no'. It does not change any *default* behaviour -- the behaviour only changes when the flag is provided.

Reply:
Yeah, you're right. Changed.